src/test/resources/pages/instructorHomeCourseDeleteSuccessful.html

```diff
@@ -1,132 +1,125 @@
-
-   <html>
-      <head>
-      </head>
-      <body>
-         <div class="container" id="mainContent">
-            <div id="topOfPage">
-            </div>
-            <h1>
-               Instructor Home
-            </h1>
-            <br/>
-                        <div class="well well-plain">
-               <div class="row">
-                  <div class="col-md-12">
-                     <form action="/page/instructorSearchPage?user=CHomeUiT.instructor.tmms" method="get" name="search_form">
-                        <div class="input-group">
-                           <input class="form-control" data-original-title="Search for student's information, e.g. name, email" data-placement="top" data-toggle="tooltip" id="searchbox" name="searchkey" placeholder="e.g. Charles Shultz, charles@gmail.com" title="" type="text"/>
-                                                      <span class="input-group-btn">
-                              <button class="btn btn-default" id="buttonSearch" type="submit" value="Search">
-                                 Search
-                              </button>
-                           </span>
-                        </div>
-                        <input name="searchstudents" type="hidden" value="true"/>
-                                                <input name="searchcommentforstudents" type="hidden" value="false"/>
-                                                <input name="searchcommentforresponses" type="hidden" value="false"/>
-                                                <input name="user" type="hidden" value="CHomeUiT.instructor.tmms"/>
-                                             </form>
+   <div class="container" id="mainContent">
```

Comment:
Hey, any reason why this html has changed from a full html check to mainContent check?